intro-to-R-tidyverse/03-intro_to_r_tidyverse_exercise.Rmd

```diff
@@ -0,0 +1,189 @@
+---
+title: 'Exercise: Clean Metadata and Prepare Gene Expression Matrix for Downstream
+  Analysis'
+output:
+  html_document:
+    df_print: paged
```

Comment:
What's this for?

Reply:
`data.frames` are paged by default, so this argument isn't necessary. Probably doesn't hurt anything, but having this here does make it different than the rest of the RMarkdowns.